splash/qtrender_lua.py

```diff
@@ -783,8 +797,8 @@ def start(self, lua_source, sandboxed, lua_package_path,
 
         try:
             main_coro = self.get_main(lua_source)
-        except (ValueError, lupa.LuaSyntaxError, lupa.LuaError) as e:
-            raise ScriptError("lua_source: " + repr(e))
+        except Exception as e:
```

Comment:
Why is a more broad except clause needed? 
Previously exceptions not from a list were propogated to the caller. I'm not sure caller was handling them though :)